R/data_transformations.R

```diff
@@ -0,0 +1,27 @@
+# This file contains functions that prepare single or multiple output columns 
+# for higher level functions such as sparc_scores_at_index. 
+
+
+#' extract_consent
+#'
+#' Extract consent data from a table with demographic information.
+#' 
+#' This function is based on sparc_summary_at_index_v2.R other similar versions may behave differently.
+#' In contrast to the original. Any patient with a unique date on consent but multiple dates of withdrawal will be reported multiple times.
```

Comment:
@AngermannBR I do not see any patients like this for SPARC. Can you please send me the master patient id?

Reply:
@ccf-tfehlmann sorry if the comment was unclear, I did not mean to imply that this is the case in the current SPARC data. This is simply highlighting that this version would behave differently should this situation arise in the future.